addons/pos_sale/migrations/13.0.1.0/post-migration.py

```diff
@@ -3,6 +3,25 @@
 from openupgradelib import openupgrade
 
 
+def _map_crm_team_id(env):
+    """From now on, every pos.order will have stored its crm.team relation, so if it's
+    changed later, those sales don't go to the new team. Anyway we have to map it in
+    the old orders so we don't loose that info in the sales report."""
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE pos_order po
+        SET crm_team_id = pc.crm_team_id
+        FROM pos_config pc, pos_session ps
+        WHERE
+            ps.id = po.session_id
+            AND pc.id = ps.config_id
```

Comment:
```suggestion
        FROM pos_session ps
        JOIN pos_config pc ON pc.id = ps.config_id
        WHERE
            ps.id = po.session_id
```
better look.

Reply:
Isn't the list of tables in the `FROM` do that very same implicitly? I mean, I tested it with real data and it's working :slightly_smiling_face: 

```
[ WITH [ RECURSIVE ] with_query [, ...] ]
UPDATE [ ONLY ] table_name [ * ] [ [ AS ] alias ]
    SET { column_name = { expression | DEFAULT } |
          ( column_name [, ...] ) = [ ROW ] ( { expression | DEFAULT } [, ...] ) |
          ( column_name [, ...] ) = ( sub-SELECT )
        } [, ...]
    [ FROM from_item [, ...] ]
    [ WHERE condition | WHERE CURRENT OF cursor_name ]
    [ RETURNING * | output_expression [ [ AS ] output_name ] [, ...] ]
```

Reply:
yeah, yeah, your query is not wrong. just that IMO, with to join looks better. aesthetics!

Reply:
Let's keep it this time as is for not relaunching CI.